tests/phpunit/AutoReview/ProjectCode/ProjectCodeTest.php

```diff
@@ -55,12 +55,11 @@
  */
 final class ProjectCodeTest extends TestCase
 {
+    /**
+     * @requires OSFAMILY Windows Cannot check if the file is executable on Windows
```

Comment:
Don't know if it's safe to comment on the annotation line. It's probably not future-proof at the very least. Might be better to comment on preceding lines instead.

Reply:
Unless PHPUnit goes from a "non-standard" annotation system, this should work fine, at least it's confirm to the PHP annotation PSR-5 spec for what it's worth.

If it breaks in the future, we can safely update as well it's not a critical part; I personally prefer to have it on the same line as it makes it clear it belongs to this specific mention. That said I don't feel strongly about it either so if others disagree I'm happy to adjust the code